addons/auth_totp/__manifest__.py

```diff
@@ -1,30 +1,36 @@
 {
-    'name': 'Two-Factor Authentication (TOTP)',
+    'name': '2-Step Authentication (TOTP)',
```

Comment:
Why do we rename all the "2-factor" into "2-step" ? 2FA is the official name (as [wikipedia](https://en.wikipedia.org/wiki/Multi-factor_authentication) is official), it is how the apps are marketed (e.g. [Authy](https://play.google.com/store/apps/details?id=com.authy.authy), [LastPass](https://play.google.com/store/apps/details?id=com.lastpass.authenticator),...)

Reply:
because @LNAodoo :D 
But I agree with you @mart-e

Reply:
A regular login is already up to 3 steps, so calling this "2-step" makes no sense.